css/_polls.scss

```diff
@@ -291,6 +291,10 @@ ol.poll-result-list {
     h1, strong ,span {
         color: #fff;
     }
+
+    button > span {
```

Comment:
Out of curiosity, why is this necesssary?

Reply:
Because the new button component uses a span and without this the text color will be overwritten bu the code above this